acceptance/tests/options/config_file/trace.rb

```diff
@@ -30,12 +30,13 @@
         on(agent, "rm -f '#{custom_fact}'")
       end
 
-      config_dir = agent.tmpdir("config_dir")
-      config_file = File.join(config_dir, "facter.conf")
-      create_remote_file(agent, config_file, config)
+      facter_conf_default_dir = get_default_fact_dir(agent['platform'], on(agent, facter('kernelmajversion')).stdout.chomp.to_f)
+      facter_conf_default_path = "#{facter_conf_default_dir}/facter.conf"
+      on(agent, "mkdir -p '#{facter_conf_default_dir}'")
+      create_remote_file(agent, facter_conf_default_path, config)
```

Comment:
This file should be reset to it original state (deleted in this case) as part of a teardown step to ensure that a modified file does not leak into other tests.